README.md

```diff
@@ -1,65 +1,56 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Math magicians
 
-> One paragraph statement about the project.
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
 
-![screenshot](./app_screenshot.png)
+- Make simple calculations.
+- Read a random math-related quote.
+
+### Screenshot
 
-Additional description about the project and its features.
+![screenshot](./app_screenshot.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- React
+- CSS
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+[Check here the Live Demo!](https://shadyshawkat.github.io/Math-magicians/)
 
 ### Prerequisites
 
+- npm or yarn
+- git
+- A text editor
+- A recent version of a browser (tested in Firefox and Chrome)
+
 ### Setup
 
-### Install
+1. Open a command line
+2. Clone the project to your computer with `git clone git@github.com:ShadyShawkat/Math-magicians.git`
+3. Go into the folder to the project with `cd Math-magicians`
+4. Build the node modules with `npm install`. 
 
 ### Usage
 
-### Run tests
-
-### Deployment
-
-
+Any change to the project should be done in the folder `src`.
 
 ## Authors
```

Comment:
- [ ] OPRIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.